src/app/react-demos/use-effect/page.tsx

```diff
@@ -0,0 +1,41 @@
+"use client"
```

Comment:
NextJs need add `use client` to make it to be a client side components. So that you can use hooks in this component.